<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Клиент</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Информация о параметрах RSA</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>p:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lb_p">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>q:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lb_q">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>n:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lb_n">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>φ(n):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lb_fn">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>e:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lb_e">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>d:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="lb_d">
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGenerateNewRSAParameters">
        <property name="text">
         <string>Сгенерировать новые параметры</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Закрытый ключ</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_privete_n"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>d:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_privete_d"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>n:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="btnReset">
           <property name="text">
            <string>Сброс</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Открытый ключ</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>e:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>n:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_public_n"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_public_e"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="btnSendPublicKey">
           <property name="text">
            <string>Отправить открытый ключ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Шифрованный текст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="tbCodedText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Открытый текст</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="tbExplicitText"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnDecrypt">
        <property name="text">
         <string>Расшифрованный текст</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnGenerateNewRSAParameters</tabstop>
  <tabstop>le_privete_d</tabstop>
  <tabstop>le_privete_n</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>le_public_e</tabstop>
  <tabstop>le_public_n</tabstop>
  <tabstop>btnSendPublicKey</tabstop>
  <tabstop>tbCodedText</tabstop>
  <tabstop>tbExplicitText</tabstop>
  <tabstop>btnDecrypt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
